tests/e2e/specs/activate-and-setup/setup-wizard.test.js

```diff
@@ -15,7 +15,6 @@ async function stepIsComplete( label ) {
 		'.woocommerce-stepper__step.is-complete',
 		{
 			text: label,
-			timeout: 5000,
```

Comment:
Looks like without this it defaults to 500ms

Reply:
Yeah it does although it shouldn't really. Actually most of the assertions have a 500ms default timeout and the timeout that is set by the woocommerce library isn't respected. It seems that there is a bug in `jest-puppeteer` library:
https://github.com/smooth-code/jest-puppeteer/issues/324

Added a workaround in 57b565404cd6569c4cdacae0ac318b12c2f3c8a7

I believe that there is also an issue with the `toClick` which causes test instability. I'll continue investigating.